language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/scopeproviders/TopLevelScopeProvider.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Check for license headers in this file and other places

Reply:
Class has been moved to a new module.